.gitignore

```diff
@@ -93,3 +93,6 @@ ENV/
 *.pth
 # Plots
 *.html
+
+# OSX related
```

Comment:
[Global .gitignores](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore) are the way to go here.

Reply:
ahh, I wasn't aware of *that* feature as well. Thanks!